include/wlr/types/wlr_input_method_v2.h

```diff
@@ -30,6 +30,8 @@ struct wlr_input_method_v2_state {
 	struct wlr_input_method_v2_delete_surrounding_text delete;
 };
 
+struct wlr_input_method_keyboard_grab_v2;
```

Comment:
This forward declaration is unnecessary.

Reply:
Removed, done.